tests/test_server.py

```diff
@@ -0,0 +1,167 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os.path
+import unittest
+import uuid
+
+try:
+    import lsst.daf.butler.server
+except ImportError:
+    pass  # Import below will set skip variable.
+
+try:
+    from fastapi.testclient import TestClient
+except ImportError:
+    TestClient = None
```

Comment:
Can you do all optional imports in one `try` block?

Reply:
Yes, probably, since the failure mode in both cases is not being able to import fastapi and/or httpx.